metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/topology/ParserTopologyBuilder.java

```diff
@@ -268,29 +266,14 @@ private static ParserBolt createParserBolt( String zookeeperUrl,
                                               Optional<String> securityProtocol,
                                               ParserConfigurations configs,
                                               Optional<String> outputTopic) {
-
-    Map<String, ParserComponents> parserBoltConfigs = new HashMap<>();
+    Map<String, WriterHandler> writerConfigs = new HashMap<>();
     for( Entry<String, SensorParserConfig> entry : sensorTypeToParserConfig.entrySet()) {
       String sensorType = entry.getKey();
       SensorParserConfig parserConfig = entry.getValue();
-      // create message parser
```

Comment:
This should not be done here because it can potentially cause serialization problems:  https://issues.apache.org/jira/browse/METRON-1793.  MessageParser and MessageFilter objects should be created during initialization and not when the container is created (and before the ParserBolt is created).  We should be doing this when prepare is called on the ParserBolt.